docs/examples/response/webhook-item-update.json

```diff
@@ -0,0 +1,76 @@
+{
+    "version": "3.28.0.7",
+    "host": "your-company.checkfront.com",
+    "type": "item",
+    "action": "update",
+    "item": {
+        "unit": "30M",
+        "item_id": "test-item",
```

Comment:
Is this really the item ID in the response? This looks more like it would be the sku? The XML response below has the correct item ID and sku fields.